### DISCUSSION TITLE
Answering the so-what question.

### DISCUSSION THREAD

**Discussion by tmcclell** [Green-Software-Foundation/oc#18] [General]
In this day and age where the industry is going through a "tech purge" and sustainability efforts is getting cut or stalled in major corporations - reference this article below

https://www.linkedin.com/search/results/content/?fromMember=%5B%22ACoAAAAXZ9kBvOKDLnHJIX2OJnlSv8jFaSx9kI8%22%5D&heroEntityKey=urn%3Ali%3Afsd_profile%3AACoAAAAXZ9kBvOKDLnHJIX2OJnlSv8jFaSx9kI8&keywords=adrian%20cockcroft&position=0&searchId=69e37de1-fb80-40e2-9af8-372c7ee5cd17&sid=2s%40&update=urn%3Ali%3Afs_updateV2%3A(urn%3Ali%3Aactivity%3A7035729287030730752%2CBLENDED_SEARCH_FEED%2CEMPTY%2CDEFAULT%2Cfalse)

All the great work we've been doing is not lost of me, but how do we help corporations identify the "value" of carbon reduction efforts. Can we answer the - "Yea great work, but ....  so what"?

@buchananwp

**Comment by aecurrie** [comment]
Ouch. He does have the answer in the article. We need AWS customers to be
demanding reporting tools. We don't need them to use those tools yet, we
just need them to demand them, which is remarkably low effort for these
enterprises. It's an email. And we can say that the tools are better on
Azure and GCP. AWS is falling behind, which is not a good look for them

**Comment by Sealjay** [comment]
So I can't be on next week - staying away from the altruistic angle of this being the "right thing to do" for the sake of the planet itself, the climate crisis, and our own future livable environment... I can see significant pragmatic answers around cost/operational efficiency.

We produced some related research here on what our stakeholders were saying.
https://www.avanade.com/en/solutions/sustainability/esg-sustainability-report

In addition, there will be increasing regulatory pressure, which will bring many software services in scope.

I'm not in the policy WG, but looking at this EU announcement:
- https://www.consilium.europa.eu/en/press/press-releases/2022/12/01/council-adopts-position-on-due-diligence-rules-for-large-companies/
- And [then the greater detail](https://data.consilium.europa.eu/doc/document/ST-15024-2022-REV-1/en/pdf), page 15 is most interesting to me. (No longer a lawyer though, so take your own legal advice, etc etc)

**Comment by tmcclell** [comment]
I asked chatGPT some questions. **Can green software strategies impact our net zero carbon goals?**

Yes, green software strategies can play a significant role in achieving our net-zero carbon goals. Green software refers to the use of software engineering practices and technologies to reduce the environmental impact of software development, deployment, and usage. 

The goal of green software is to minimize the carbon footprint of software by reducing energy consumption, minimizing waste, and improving the efficiency of software development and usage.

Here are some examples of how green software strategies can help achieve net-zero carbon goals:

Energy-efficient software design: By using software engineering practices like optimizing code, reducing network traffic, and reducing server loads, energy consumption can be reduced. This can be achieved by developing software that consumes fewer resources and runs more efficiently.

Cloud computing: Cloud computing can help reduce energy consumption by allowing companies to share computing resources, thereby reducing the need for physical hardware. This means that companies can save on energy costs while reducing their carbon footprint.
Virtual meetings: With the use of video conferencing tools, remote collaboration, and other digital tools, companies can reduce the need for physical travel, which contributes to carbon emissions. By using digital tools, employees can work from anywhere in the world, reducing the need for business travel.

Smart buildings: Software can be used to optimize building systems, such as heating, ventilation, and air conditioning (HVAC), to reduce energy consumption. Smart building technology can also help reduce the carbon footprint of buildings by optimizing energy usage and reducing waste.

In conclusion, green software strategies can play a crucial role in achieving net-zero carbon goals by reducing energy consumption, minimizing waste, and optimizing efficiency in software development and usage. By adopting green software practices, companies can reduce their carbon footprint and contribute to a more sustainable future.

**What metrics can we use to determine if green software is impacting our climate goals?**
- Energy consumption
- Carbon emissions
- Resource effiency
- Waste reduction
- Renewable energy use
- User behavior: The impact of software on the behavior of users can also be measured. For example, if a software application encourages users to use public transportation instead of driving, it could have a positive impact on climate goals.

By measuring these metrics, we can assess the impact of green software on our climate goals and identify areas for improvement.

**So what if we don't reach out net zero carbon goals?**

If we do not reach our net-zero carbon goals, we will continue to see the negative effects of climate change, such as rising global temperatures, increased frequency and intensity of natural disasters, and rising sea levels. These impacts can lead to a range of environmental, economic, and social consequences, including:

1. Environmental consequences: Climate change can lead to the loss of biodiversity, the degradation of ecosystems, and the depletion of natural resources. This, in turn, can have significant consequences for food security, water availability, and human health.

2. Economic consequences: Climate change can also have significant economic consequences, including damage to infrastructure, reduced agricultural productivity, and increased healthcare costs.

3. Social consequences: Climate change can also exacerbate existing social inequalities, with marginalized communities and vulnerable populations bearing the brunt of its impacts.

**Overall, failing to reach net-zero carbon goals could have catastrophic consequences for our planet and its inhabitants. It is therefore essential that we continue to work towards achieving these goals and take meaningful action to mitigate the effects of climate change.**

**Comment by jawache** [comment]
@tmcclell our state of green software report should got part way to helping answer this question, it's a common question, there are regulatory, market forces, employee forces to be used as arguments. I would come back to this in 2 months time once the report is released, circulate those internally and use the feedback to direct the focus of next years survey and report to help plug any gaps.

**Comment by PindyBhullar** [comment]
The "so what question" is important for organisations to answer. For example the SCI calculations, once we have a SCI number for an organisation what would developers, engineers, businesses then do with that number. What are the key decisions they can make and what levers would they need to pull? 

I am very interested in this topic as we need to also take a broader view of how green software can influence the wider organisation for real behavioural change. 

I have been speaking to Tammy about my research and it fits in nicely with the points raised in this discussion. Can I suggest I pull together a proposal for further discussion so we can kick start some discussions and actions?

**Comment by aecurrie** [comment]
Count me in on this. I had a person asking me yesterday if there was a
green maturity matrix from the GSF and I think it's a really good idea. The
maturity matrix is related to this "what are the steps?" question I
think. It would be great if we produced one. Maybe a new project? Happy to
propose/volunteer.

I think measurement is probably quite far along in the matrix TBH

**Comment by Sealjay** [comment]
Do we have a Google doc to work from? Or we could do it in this repo, in a pull request

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by PindyBhullar** [comment]
I will create a doc that we can review and comment.

**Comment by PindyBhullar** [comment]
Project Proposal
Developing green software is important to reduce the overall greenhouse gas emissions, and in particular Co2. Emissions contributed by technology have been widely discussed by industry bodies, academics and inter-governmental organisations. It is also clear that during development and operational use of applications requires measuring of Co2 to create baseline metrics and enable engineers and architects to consider options to reduce emissions. This cycle is continuous and the ability to get to zero emissions from using software may not be achievable for some organisations as there are various factors that can influence emissions, for example where the application is deployed and the energy it consumes, there are many more!
The development of the SCI provides a mechanism to calculate software carbon emissions. But the “so what” question is still outstanding and needs addressing. 
This proposal is to formulate a project that tackles the broader organisation adoption of calculations and the steps organisations require to ensure measuring of Co2 is a key component of application development. 
I suggest we use the TCFD framework as a basis to integrate areas of the organisation that enables Co2 data to flow into strategic and executive level decisions. 
The proposal will require the output from the various GSF WG’s to feed into this project. The assumption that measuring from cloud and on prem data centres, and techniques to measure the types of energy used into these premises is out of scope for this proposal. This proposal is to take those outputs and provide a roadmap/framework for organisations to use and apply to make real differences. This will encompass, shared data principles, development of OKR’s and KPI’s, changing behaviours, embedding into organisational change processes and practices.
A list of some questions we could start to address:
1.	Once the baseline of Co2 emissions from software has been established how do organisations make meaningful decisions?
2.	How do organisations set targets using Co2 emissions data from software?
3.	What are the key behavioural changes and influences required in the devops community to adopt green principles in engineering practices?
4.	How can the procurement teams leverage from software emissions data, to ensure contracts with ITO/BPO vendors comply with organisational targets?
5.	What are they key decisions and choices during business case development for software solutions?
6.	How do the costs, business growth, and Co2 targets interplay in an organisation?
7.	How can organisations set meaningful OKR’s and KPI’s to reduce Co2 from software usage?
8.	What are the challenges organisations face – Is there an opportunity for industry, and tech innovators help to alleviate the challenges? 
9.	What impact does the Co2 emissions have on regulatory disclosures? What’s the timeline for granular reporting?
Thoughts?

**Reply by jawache** [reply]
Hi @PindyBhullar great project proposal! Would you mind creating a project proposal request using this link: https://github.com/Green-Software-Foundation/oc/issues/new?assignees=tmcclell%2C+Sealjay%2C+seanmcilroy29&labels=project%2C+proposed&projects=&template=new-project-proposal.md&title=%5BProject%5D+Title+Of+Project

**Comment by aecurrie** [comment]
I'm saying that the purpose of moving up the levels of the maturity matrix is not to be ethical or nice. It is a risk management strategy to provide resilience to:
Tightening ESG and other (e.g. EU) reporting requirements around sustainable IT. 
Physical disruption to IT systems caused by climate change.
Pricing changes, for example the impact of dynamic tariffs or reuse, repair and recycle rules from the EU. 
What else? Other risks?
